gsuite/resource_group.go

```diff
@@ -63,6 +64,22 @@ func resourceGroup() *schema.Resource {
 	}
 }
 
+func validateEmail(v interface{}, k string) (warnings []string, errors []error) {
+	if v == nil || v.(string) == "" {
+		return
+	}
+	email := v.(string)
+	// TODO: This does not account for non-standard but still RFC compliant
+	// email addresses such as "some@email"@domain.com
+	local := strings.Split(email, "@")[0]
+	if len(local) > 63 {
+		errors = append(errors,
+			fmt.Errorf("local portion of email %s exceeds 63 characters", email))
+	}
+
+	return
+}
+
```

Comment:
Do you think this length restriction applies to users as well? Thinking if we should move this to utils and add to users' `ValidateFunc` as well

Reply:
There's definitely a few places that have email values as inputs.  Happy to move and expand if it's helpful.

Reply:
Definitely, if you don't mind

Reply:
sure thing

Reply:
k, did a second pass at this.  Moved the validation function to utils, added the validation to more resources, and added some unit tests to help make it clear the types of valid email formats expected.